hudi-spark-datasource/hudi-spark/pom.xml

```diff
@@ -188,6 +188,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.calcite</groupId>
+      <artifactId>calcite-core</artifactId>
```

Comment:
we can remove the dependence calcite in spark? can use `Math.abs` in TestParquetColumnProjection right?

Reply:
yeah, we better not introduce Calcite dependency which may incur jar conflicts.

Reply:
This makes more sense, I'm modifying this PR to switch to use `scala.math.abs`